behind-the-veil-siteroot/imports/ui/components/pages/home/HomePage.jsx

```diff
@@ -1,22 +1,201 @@
+    };
+
     return (
         <WhiteBackground pageLayout={PageLayout.LARGE_CENTER}>
-            <span>Home Page to be done!!</span>
+            {/* Container for the images and the left and right buttons for sort through images */}
+            <div className="inline-flex items-center justify-center h-[25rem] md:h-[30rem]">
+                <div className="hidden md:flex md:items-start md:justify-center md:w-1/6">
+                    <Button onClick={goToPreviousImage} className="bg-white">
+                        <ChevronLeftIcon className="size-10 stroke-[4]" />
+                    </Button>
+                </div>
+                <div className="inline-flex justify-center w-full md:w-4/6 h-full">
+                    <div className="hidden lg:flex lg:items-start lg:justify-end lg:h-full">
+                        <img
+                            src={
+                                images[
+                                    currentImageIndex === 0
+                                        ? images.length - 1
+                                        : currentImageIndex - 1
+                                ]
+                            }
+                            className="object-cover h-96 w-72 rounded-[2rem] brightness-50 -mr-16 shadow-2xl"
+                        />
+                    </div>
+                    <div className="flex md:items-center h-full">
+                        <img
+                            src={images[currentImageIndex]}
+                            className="object-cover w-full h-96 md:w-96 rounded-[2rem] z-10 shadow-2xl"
+                        />
+                    </div>
+                    <div className="hidden lg:flex lg:items-start lg:h-full">
+                        <img
+                            src={
+                                images[
+                                    currentImageIndex === images.length - 1
+                                        ? 0
+                                        : currentImageIndex + 1
+                                ]
+                            }
+                            className="object-cover h-96 w-72 rounded-[2rem] brightness-50 -ml-16 shadow-2xl"
+                        />
+                    </div>
+                </div>
+                <div className="hidden md:flex md:items-start md:justify-center md:w-1/6">
+                    <Button onClick={goToNextImage} className="bg-white">
+                        <ChevronRightIcon className="size-10 stroke-[4]" />
+                    </Button>
+                </div>
+            </div>
+            {/* Container for left and right buttons to sort through images when screen is smaller than medium size*/}
+            <div className="inline-flex flex-row items-center justify-center md:hidden">
+                <div className="flex items-start justify-center w-[45%]">
+                    <Button onClick={goToPreviousImage} className="bg-white">
```

Comment:
small nitpick, are these buttons supposed to have transparent background color? bg-white makes them stand out a bit from the background 
![image](https://github.com/Monash-FIT3170/Behind-The-Veil/assets/95511987/5dd28a37-d29a-42fd-92a6-9cac55ac72c6)

Reply:
Yeah, I can fix it though

Reply:
Fixed to make them transparent